packages/contracts/buidler-env.d.ts

```diff
@@ -0,0 +1,2 @@
+/// <reference types="@nomiclabs/buidler-ethers/src/type-extensions" />
+/// <reference types="@nomiclabs/buidler-waffle/src/type-extensions" />
```

Comment:
Just want to make sure these are supposed to be commented out

Reply:
Yeah it's just the weird way that typescript has you explicitly identify typing files (https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html).